qiskit_ibm_runtime/accounts/storage.py

```diff
@@ -27,7 +27,7 @@ def save_config(
     logger.debug("Save configuration data for '%s' in '%s'", name, filename)
     _ensure_file_exists(filename)
 
-    with open(filename, mode="r", encoding="utf-8") as json_in:
+    with open(filename, encoding="utf-8") as json_in:
```

Comment:
Why does ruff bother to make the effort to make such a change? It was even more readable before

Reply:
Do you see what's the confusion that they're trying to avoid? And why does this confusion occur specifically with open modes, and not every time that some function parameter is set to its default value? Anyway if that's ruff's decision then let it be.